src/components/InputTodo.jsx

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable import/no-extraneous-dependencies */
+/** @format */
+import { useState } from 'react';
```

Comment:
- [ ] Kindly make sure you do not disable linters in your code because they are meant to check for best practices which helps in making you a better developer in following the industry’s standard of writing clean code 👍.